loop/routes/validators.js

```diff
@@ -223,6 +224,95 @@ module.exports = function(conf, logError, storage) {
   }
 
   /**
+   * Validate the room status update fields passed in the body.
+   **/
+  function validateRoomStatusUpdate(req, res, next) {
+
+    if (req.body.action && req.body.action === 'status') {
+
+      // Mandatory fields
+      var expectedFields = ['event', 'state', 'connections', 'sendStreams',
+        'recvStreams'
+      ];
+
+      var field;
+
+      for (var i = 0; i < expectedFields.length; i++) {
```

Comment:
Unless I'm wrong, with forEach, I can't return out of the function (``break``) on the first error